cmd/agent/subcommands/run/command_windows.go

```diff
@@ -13,6 +13,7 @@ import (
 	_ "expvar"         // Blank import used because this isn't directly used in this file
 	_ "net/http/pprof" // Blank import used because this isn't directly used in this file
 
+	"github.com/DataDog/datadog-agent/comp/agent/containertagger"
```

Comment:
```suggestion
	"github.com/DataDog/datadog-agent/comp/agent/cloudfoundrycontainer"
```

Reply:
the folder was renamed to `cloudfoundrycontrainer` but not the package name, it's still `containertagger`.